flare_portal/experiments/forms.py

```diff
@@ -68,6 +71,67 @@ def save(self, *, experiment: Experiment) -> None:
         )
 
 
+class ParticipantUploadForm(forms.Form):
+    import_file = forms.FileField(
+        validators=[FileExtensionValidator(["csv"])],
+    )
+
+    @staticmethod
+    def open_csv(field):
+        # Open uploaded file
+        data = io.StringIO(field.read().decode("utf-8"))
+        return csv.DictReader(data)
```

Comment:
I'm not sure it's worth extracting this because it's used only once and it's used only within this class. Besides it's only 2 lines long. Probably worth just inlining this bit.

Reply:
Done.